### PR TITLE
G protein structure statistics page update

### PR DESCRIPTION
**Pull request by Jimmy-A-Caroli**
**Major updates:**
- created 'structure/gprot_statistics' page, featuring statistics about G protein related structures
- set a variable for identification of page (structure/statistics and structure/gprot_statistics) in the view
- unrelated: applied same concept to ligand and ligand bias statistics page, removing redundant code
- page contains two tables illustrating all the available G protein structures and a detailed view on the GPCR-G protein complexes
- tree graphs show GPCR-G protein complexes coverage (external dots)

### CODE REVIEW

File: home/templates/home/mainmenu_gpcr.html [hunk 1]
```diff
@@ -158,6 +158,12 @@
         <!-- STRUCTURES -->
         <li class="divider"></li>
         <li class="nav-header disabled text-left"><a>STRUCTURES</a></li>
+        <li><a href="/structure/gprot_statistics">Structure statistics
+            <button mp4="https://files.gpcrdb.org/video/structures1.mp4" type="button"
```
Comment (AJK-dev) [inline home/templates/home/mainmenu_gpcr.html:162]: Can you remove the video button and icon? It's not for this specific page.

File: structure/views.py [hunk 2]
```diff
@@ -506,31 +507,53 @@ class StructureStatistics(TemplateView):
...
 		for f in families:
 			lookup[f.slug] = f.name
 
 		all_structs = Structure.objects.all().prefetch_related('protein_conformation__protein__family')
+		noncomplex_gprots = SignprotStructure.objects.filter(protein__family__slug__startswith='100')
 		all_complexes = all_structs.exclude(ligands=None)
-
 		all_gprots = all_structs.filter(id__in=SignprotComplex.objects.filter(protein__family__slug__startswith='100').values_list("structure__id", flat=True))
+		###### these are query sets for G-Prot Structure Statistics
+		all_g_A_complexes = all_gprots.filter(protein_conformation__protein__family__slug__startswith='001')
```
Comment (AJK-dev) [inline structure/views.py:524]: Not for now, but it would be nice if this could be a bit more concise

**Review by AJK-dev — CHANGES_REQUESTED** [review]
Looking good, if you could remove the video link and icon. I'll merge it!

File: structure/views.py (hunk 2 above)
Comment (Jimmy-A-Caroli) [reply structure/views.py:524]: I will mark it as a ToDo while working on the upcoming tweaks

**Review by AJK-dev — APPROVED** [review]
Minor star import complaint from Codacy, but otherwise 👌

**Pull request merged**